userapi/api/api.go

```diff
@@ -30,6 +32,18 @@ type UserInternalAPI interface {
 	QueryAccountData(ctx context.Context, req *QueryAccountDataRequest, res *QueryAccountDataResponse) error
 }
 
+// InputAccountDataRequest is the request for InputAccountData
+type InputAccountDataRequest struct {
+	UserID      string          // required: the user to set account data for
+	RoomID      string          // optional: the room to associate the account data with
+	DataType    string          // optional: the data type of the data
```

Comment:
This is not optional according to the code

Reply:
You're right - that's a mistake in the comment.